xoinvader/ingame.py

```diff
@@ -251,9 +249,9 @@ def events(self):
 
     def update(self):
         self._collision_manager.update()
-        self.wave.update()
-        if not self.wave.running:
-            self.wave.start()
+        self.level.update()
```

Comment:
It's quite obvious for me that levels are GameObjects that follow same rules but okey, we will return to this conversation later.